docs/source/quickstart.rst

```diff
@@ -29,7 +29,7 @@ For more details on docker setup, please refer to `docker readme <https://github
 
 - For Windows, due to the lack of support for Jax, we recommend to use docker or WSL, please refer to `TC via windows docker <contribs/development_windows.html>`_ or `TC via WSL <contribs/development_wsl2.html>`_.
 
-- For Mac with M series chips (arm architecture), please refer to `TC on Mac M series <contribs/development_MacARM.html>`_.
+- For MacOS, please refer to `TC on Mac <contribs/development_Mac.html>`_ or `在Mac上安装TC <contribs/development_Mac_cn.html>`_.
```

Comment:
dont add chinese translation link here.

Reply:
Where should I add it?

Reply:
> Where should I add it?

maybe in your mac english installlation doc

``for chinese version of the doc, see link``

Reply:
fixed.

Added it to translation file.